### ISSUE TITLE
Alternative Soundtrack

### ISSUE THREAD

**Issue by r4dh4l** [Anuken/Mindustry-Suggestions#4926]
### Describe the content or mechanics you are proposing.

I really like the game but the soundtrack is a little bit melancholic in my taste (but excellent quality). There is some electronic CC music which could fit composed by Timecrawler 82 which could be added as alternative soundtrack:
- https://timecrawler82.bandcamp.com/album/infinity
- https://timecrawler82.bandcamp.com/album/osaka-lights

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

An alternative soundtrack could be added as opt-in option so the change would only affect people who would like to a different background music. Nothing would change for players who like the current game music.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by Leo-MathGuy** [comment]
Maybe anuken should make a music-submittion contest like he is doing with campaign maps right now?

**Comment by itcannotbe** [comment]
> Maybe anuken should make a music-submittion contest like he is doing with campaign maps right now?

This suggestion is problematic, because it would hard for the dev to filter the submission for originality in order not to get run ins with other people's copyright

**Comment by Leo-MathGuy** [comment]
What I meant, is like music composing competition, not just copy the music. Maybe for originality checking, they could send the source file (from whatever DAW they used) to a moderator to confirm originality.

**Comment by itcannotbe** [comment]
Licensing isn't the issue, it reuse of music or its components there of.

Same way artist sue other artist for similarities (even for a few seconds)

**Comment by S0meM** [comment]
Yes, this is the answer for a lot of these problems, but make a mod. I don't think an alternate soundtrack for the game would be a good idea. Or just mute mindustry and play a soundtrack yourself.

**Comment by r4dh4l** [comment]
> I don't think an alternate soundtrack for the game would be a good idea. Or just mute mindustry and play a soundtrack yourself.

Sure, playing music in background is always possible but @S0meM you know that's not the same.

> Licensing isn't the issue, it reuse of music or its components there of.

@itcannotbe I don't understand your point. Why licensing isn't a problem? I thought (compatible) CreativeCommons music would be a minimum requirement to add additional music to a FLOSS game.
